contracts/StableJoeStaking.sol

```diff
@@ -95,20 +99,24 @@ contract StableJoeStaking is Initializable, OwnableUpgradeable {
      * @param _joe The address of the JOE token
      * @param _feeCollector The address where deposit fees will be sent
      * @param _depositFeePercent The deposit fee percent, scalled to 1e18, e.g. 3% is 3e16
+     * @param _smolJoes The address of ERC721 Smol Joes NFT
      */
     function initialize(
         IERC20Upgradeable _rewardToken,
         IERC20Upgradeable _joe,
         address _feeCollector,
-        uint256 _depositFeePercent
+        uint256 _depositFeePercent,
+        IERC721Upgradeable _smolJoes
```

Comment:
Add a setter for smolJoe's address as we can't use initialize during an upgrade as the contract has already been initialized.